### ISSUE TITLE
As a supervisor, I want more clarity on which conversation template to use.

### ISSUE THREAD

**Issue by Travis-A-Clark** [bcgov/performance#395]
We need a design that makes it clear that the Performance Check-In template is the default selection. Let's remove the search box to save space and change the format: 

"Select a template below to set up a performance conversation. Templates include information on how to prepare for the conversation, questions to consider when having the conversation, and an attestation and sign-off area to formalize the results.

The Performance Check-In template can be used in most situations. It includes options to capture progress against goals, celebrate successes, discuss ways to improve future performance outcomes, and record an overall performance evaluation. 

**Insert Performance Check-In template here**

The other templates can be used as required to support conversations that require a more specific focus. Select a topic below to read more in the <i>When to Use this Template</i> section."

**Insert remaining templates here**

This design should be coded into all areas where the templates are listed.

**Comment by phoebesolomon** [comment]
![Conversation Templates Page Design.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/71997563-544d-47ff-b356-c346d7ebec23)

**Comment by Travis-A-Clark** [comment]
Hey @phoebesolomon ! Can we still keep the original description above the Performance Check-In header?

"Select a topic below to set up a performance conversation. Templates include suggestions for when to select a given conversation topic, questions to consider when having the conversation, and an attestation and sign-off area to formalize the results."

Also, should the Performance Check-In and Other headings be the same font/colour instead of one blue and one black?

Thanks!

**Comment by phoebesolomon** [comment]
@Travis-A-Clark I've revised based on your request. I've removed the blue colour for the 'Performance Check-in Template' header and added the description above. However, I am suggesting that we keep the blue outline as it is another way to visually indicate to the user that the Performance Check-in Template is the default selection. Perhaps we should introduce this concept when the user hovers over the other templates as well to keep it consistent. Let me know what you think. ![Conversation Templates Page Design.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/2c234c8f-1f2f-44ff-97b6-943f475c1190)
